### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by iamtheru**
- [DEMO LINK](https://iamtheru.github.io/layout_moyo-header/)
 - [TEST REPORT LINK](https://iamtheru.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+.nav__link {
+  text-decoration: none;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  color: black;
+  padding-top: 23px;
+  padding-bottom: 23px;
+  white-space: nowrap;
+
```
Comment (vkryvytskyy) [inline src/style.css:46]: remove empty lines

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+.nav__link:hover,
+.is-active {
+  color: #00acdc;
+
```
Comment (vkryvytskyy) [inline src/style.css:52]: remove empty lines

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+.is-active:after {
+  content: "";
+  border: 2px solid ;
+  display: block;
+  margin-top: 18px;
+  border-radius: 10px;
+  color: #00acdc;
+
```
Comment (vkryvytskyy) [inline src/style.css:62]: remove empty lines

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+.is-active:after {
+  content: "";
+  border: 2px solid ;
+  display: block;
+  margin-top: 18px;
+  border-radius: 10px;
+  color: #00acdc;
```
Comment (vkryvytskyy) [inline src/style.css:61]: better to make it with position: absolute

File: src/index.html [hunk 5]
```diff
@@ -1,14 +1,60 @@
...
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header class="header">
+      <a href="/">
```
Comment (vsm2k17) [inline src/index.html:17]: Використовуй # як заглушку (замість /)

File: src/style.css (hunk 3 above)
Comment (vsm2k17) [inline src/style.css:62]: delete extra line

**Pull request closed without merging**